src/Console/ValidateTableCommand.php

```diff
@@ -21,6 +21,94 @@ class ValidateTableCommand extends Command
      */
     protected $description = 'Generate a validation array for the specified table';
 
+    /**
+     * The laravel validation rule to map column types to.
+     *
+     * @var array
+     */
+    protected $columnTypeRules = [
```

Comment:
I liked it thank you, but this array seems too long, could you move it to a json file and resolve it by json_decode() method?
then I will merge it.

Reply:
Moved array data to a json file. Can also be published to the app's resources dir for customization.

Reply:
I was about to merge it but I made some changes to to .gitignore file  that caused conflicts to your PR, can you please resolve it :)

Reply:
I've fixed the merge conflict. Should be good to merge now.